### PR TITLE
Adding links for PR.gov, Code for America, and PR.gov analytics

### PR DESCRIPTION
**Pull request by ainsleywagon**
Just adding analytics code!

### CODE REVIEW

**Comment by ainsleywagon** [comment]
This is making my local server very angry though.

**Comment by ainsleywagon** [comment]
screen shot 2014-09-29 at 4 37 30 pm

**Comment by ainsleywagon** [comment]
@MrMaksimize this is ready for review - but the analytics code is throwing everything out of whack. I just took what they gave me and threw it into a jade translator, but I'm not sure how it should be formatted.

File: views/home.jade [hunk 1]
```diff
@@ -100,6 +102,14 @@ html
           ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
           var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
         })();
+       PR.gov analytics code
```
Comment (MrMaksimize) [inline views/home.jade]: Hrm.  I dont like this.  This is just straight up google analytics -- and we already have that. I don't want to give up our analytics control just yet -- I'm gonna hold this for right now till we can decide what to do and how to do it in a way that there's analytics firing for both.

Comment (ainsleywagon) [inline views/home.jade]: @MrMaksimize Okay, cool- I guess that's something to work out with them. The other analytics code was also throwing an error so I commented it out for now. I don't know what's up.

**Comment by MrMaksimize** [comment]
Yeah -- ure right @ainsleywagon 

http://take.ms/W0sPy

Do this -- Don't comment stuff out, but just bring the analytics code back to the way it was.  And then just make an issue with the code that pr.gov told to put in and we'll figure out how to do it.  

After you clean it out we can take it )

**Pull request merged**